packages/build/src/plugins_core/build_command.js

```diff
@@ -6,6 +7,16 @@ import { addErrorInfo } from '../error/info.js'
 import { getBuildCommandDescription } from '../log/description.js'
 import { logBuildCommandStart } from '../log/messages/steps.js'
 import { getBuildCommandStdio, handleBuildCommandOutput } from '../log/stream.js'
+import { getBlobsDir } from '../utils/blobs.js'
+
+async function cleanupBlobsDir(buildDir, publishDir) {
+  const blobsDir = getBlobsDir({ buildDir, publishDir })
+  try {
+    await fsPromises.rm(blobsDir, { recursive: true, force: true })
```

Comment:
```suggestion
    await rm(blobsDir, { recursive: true, force: true })
```

[dust]: use the function from the named import